### PR TITLE
Setup sbt-ci-release

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

**Review by gabro — COMMENTED** [review]
💯

File: project/PublishPlugin.scala [hunk 1]
```diff
@@ -1,72 +0,0 @@
...
-        "Jorge Vicente Cantero",
-        "jorgevc@fastmail.es",
-        url("https://jvican.github.io/")
-      ),
-      Developer(
-        "olafurpg",
-        "Ólafur Páll Geirsson",
-        "olafurpg@gmail.com",
-        url("https://geirsson.com")
-      )
```
Comment (gabro) [inline project/PublishPlugin.scala:61]: aren't these info lost?

Comment (olafurpg) [reply project/PublishPlugin.scala:61]: Yup, pushing another commit soon. Please don't merge!

Comment (gabro) [reply project/PublishPlugin.scala:61]: Hehe sure! I was just passing by 😅

**Comment by olafurpg** [comment]
OK should be good now!

**Pull request merged**